sequencer/src/api.rs

```diff
@@ -5910,7 +5912,7 @@ mod test {
             .build();
 
         let api_port = pick_unused_port().expect("No ports free for query service");
-        tracing::info!("API PORT = {api_port}");
+        println!("API PORT = {api_port}");
```

Comment:
curious about the switch to println here

Reply:
I think for tests it's slightly preferrable to use `println` so that doing something like `env RUST_LOG=error cargo test ...` only shows minimal logs plus the test output without having to write a complex filtering expression for rust log.